### ISSUE TITLE
PGP: publicKey and fingerprint for Person and Organization?

### ISSUE THREAD

**Issue by Harald-Effenberg** [schemaorg/schemaorg#1522]
Hi!

Wouldn't it be useful to add the properties "publicKey" and "fingerprint" to the schemata "Person" and "Organization"?

Regards,
Harry

**Comment by gjvnq** [comment]
I agree that this would be a good idea. The field ```publicKey``` should probably follow a format like "[System/Hash Function]:[Hex value]". Example: ```PGP:FEDB7014D7D90ADE3B74B54B96E477D4BFC6B8D9```

A more flexible option would be for PublicKey to be its own entity with fields like: ```algorithm```, ```modulus```,   ```publicExponent``` and ```url```. However, this seems to be too complicated for such little use.
